step/serialization.md

```diff
@@ -1,109 +1,553 @@
-# Serialization holder
+# Serialization
 
 ## Overview
 
-This 5-minute optional lab will show you how **`jlink`** can greatly improve the size of a Java runtime image.
+Serialization has long been a sticky subject in Java. Brian Goetz describing it as a paradox, a key to Java's success allowing for transparent remoting, but also the implementation of serialization making a number of key mistakes. 
```

Comment:
"Seriaization is an important mechanism in the Java platform. This tutorial will summarise how serialisation, and de-serialisation work. It will the introduce you the deserialization filters, introduced in JDK 9 and enhanced in JDK 17, an important mechanism to secure and overcome the typical serialisation weakness".

Reply:
PS: my browser tends to sometimes suggest "seriai**s**ation", just make sure to be consistent (Z or S).